webapi/Services/SemanticKernelProvider.cs

```diff
@@ -22,6 +23,8 @@ namespace CopilotChat.WebApi.Services;
 /// </summary>
 public sealed class SemanticKernelProvider
 {
+    private static IMemoryStore? _memoryStore;
```

Comment:
Maybe rename this guy _volatileMemoryStore since it only applies to volatile memory...